targets.cfg

```diff
@@ -80,7 +80,7 @@ minversions =
 branches = gotham,helix,isengard,jarvis,krypton,leia,matrix
```

Comment:
Would it make sense to remove gotham to leia as well, since a minimum xbmc.python version 3.0.0 will only be satisfied by matrix+ ?

Reply:
yeah, makes perfect sense.
thx!

Reply:
turned out it didn't make sense..
we do want to pull skins/scrapers/resources from lower branches.

i'll revert it.